pkg/piecestore/psserver/bucket_refresher.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package psserver
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/dht"
+	"storj.io/storj/pkg/pb"
+)
+
+// Error is a standard error class for this package.
+var (
+	Error                = errs.Class("kademlia bucket refresher error")
+	defaultCheckInterval = flag.Duration("piecestore.kbucket-refresher.check-interval", time.Hour, "number of seconds to sleep between updating the kademlia bucket")
+)
+
+// refreshService contains the information needed to run the bucket refresher service
+type refreshService struct {
+	logger *zap.Logger
+	rt     dht.RoutingTable
```

Comment:
Use `*kademlia.RoutingTable` directly.

Reply:
In this case the `dht.DHT` should eventually go away. But as a general guideline for using interfaces:

1. when there are multiple implementations (e.g. `storage.KeyValueStore`)
2. when there's an import cycle that cannot be broken (e.g. masterdb things)
3. when it makes sense to be compatible to extension (e.g. `io.Reader`, `io.Writer` in API-s)

All interfaces create an indirection which makes code harder to understand. At the moment `dht.DHT` is creating an indirection... so it's not critical to fix, but no reason to keep using it either.

Reply:
gotcha that makes sense ty

Reply:
done